### ISSUE TITLE
Nz-select "multiple" inside nz-dropdown-menu gives an error when trying to deselect items

### ISSUE THREAD

**Issue by IonelLupu** [NG-ZORRO/ng-zorro-antd#5578]
### Reproduction link
[https://stackblitz.com/edit/ng-zorro-antd-ivy-dudmbn](https://stackblitz.com/edit/ng-zorro-antd-ivy-dudmbn)

### Steps to reproduce
- add a nz-select with nzMode="multiple" inside a nz-dropdown-menu that is triggered by the click event
- open the dropdown, then the select box and select a few items
- close the select box and the dropdown by clicking twice, on the blank page
- open de dropdown and then the select box and try to remove the selected items
- observe the error in the console

### What is expected?
The items to be removed

### What is actually happening?
Getting an error:
- Chrome: Cannot read property 'zoomMotion' of undefined
- Firefox: elementStates is undefined

| Environment | Info |
|---|---|
| ng-zorro-antd | 9.2.2 |
| Browser | Chrome 84, Firefox 78 |

**Comment by vthinkxie** [comment]
Hi
I believe this a duplicate of https://github.com/NG-ZORRO/ng-zorro-antd/issues/5290
blocked by https://github.com/angular/angular/issues/24041

**Comment by IonelLupu** [comment]
Hi @vthinkxie 
Will disabling animation using `nzNoAnimation` solve this? (temporarily)
